docs/releases/1.10.txt

```diff
@@ -471,6 +471,31 @@ creates the possibility of a deadlock. To adapt your code in the case of RQ,
 you can `provide your own worker script <http://python-rq.org/docs/workers/>`_
 that calls ``django.setup()``.
 
+Timezone-aware :class:`~django.core.files.storage.Storage` API
+--------------------------------------------------------------
+
+Storage backends now present a timezone-aware API, with new methods
+``get_accessed_time()``, ``get_created_time()`` and ``get_modified_time()``
+returning timezone-aware datetime objects if :setting:`USE_TZ` is ``True``,
+and naive objects in the local timezone otherwise.
+
+Code that uses storage backends should update to use the new methods.
```

Comment:
I would move from this paragraph down into a section in the "Features deprecated" section and remove the note under "Misc".